### DISCUSSION TITLE
Wildcard certificates and --stateless mode

### DISCUSSION THREAD

**Discussion by MrRinkana** [acmesh-official/acme.sh#4953] [Q&A] [answered]
Hello!
Are wildcard certificates supported/allowed when using --stateless mode?

I was trying to issue a wildcard cert for my domain with letsencrypt_test server like so: `acme.sh --issue -d *.example.com --stateless --server letsencrypt_test` but it errors out with:

> Error, can not get domain token entry *.example.com for http-01
> [Thu 18 Jan 2024 01:58:55 PM CET] The supported validation types are: dns-01 , but you specified: http-01
> [Thu 18 Jan 2024 01:58:55 PM CET] Please check log file for more details: /var/lib/acme/.acme.sh/acme.sh.log

Which made me suspect that wildcards are not supported in stateless mode (it kind of makes sense as how would the CA know where to send the request) but I wanted to verify that its not just me doing something wrong, or if there's a way to do the equivalent (since in my case all subdomains are at the same ip, shared with example.com)..

**Comment by neilpang** [comment, marked as answer]
correct.  wildcard cert can only be validated by dns-01. 
but stateless is http-01.
